lifelines/estimation.py

```diff
@@ -1152,6 +1158,10 @@ def _newton_rhaphson(self, X, T, E, initial_beta=None, step_size=1.,
                                    include_likelihood=include_likelihood)
             # Do not override hessian and gradient in case of garbage
             h, g = output[:2]
+            if self.penalizer > 0:
+                # add the gradient and hessian of the l2 term
+                g -= self.penalizer * beta.T
```

Comment:
I most familiar with L2 reg in the context of linear/logistic regression, and typically the formula is
```
min( CostFunction(f(x), y + (Lambda * theta^2)
```
(a la [Wikipedia article](https://en.wikipedia.org/wiki/Regularization_(mathematics)))

But I might be missing something here since I'm not super familiar with the details of fitting a cox model